Add Datadog

euan-reid opened the pull request:
* [Global rank 6706](https://www.similarweb.com/website/datadoghq.com/)
* Developer tool containing metric data, APM traces, and logs from applications

euan-reid committed:
Add Datadog

Signed-off-by: Euan Reid <github@euanreid.com>

euan-reid committed:
Add Datadog icon

Signed-off-by: Euan Reid <github@euanreid.com>

euan-reid committed:
Correct syntax

hkamran80 commented on entries/d/datadoghq.com.json:
```diff
@@ -0,0 +1,13 @@
+{
+  "Datadog": {
+    "domain": "datadoghq.com",
+    "url": "https://app.datadoghq.com",
```
```suggestion
```

Because going to `https://[domain]` works, the `url` is not necessary.

euan-reid replied:
There's no ability to sign in on the top-level domain, it redirects to `app.` so it seemed more appropriate to send folks there. But happy to remove it.

hkamran80 commented on entries/d/datadoghq.com.json:
```diff
@@ -0,0 +1,13 @@
+    "tfa": [
+      "totp"
+    ],
+    "documentation": "https://docs.datadoghq.com/account_management/multi-factor_authentication/",
```
```suggestion
    "documentation": "https://docs.datadoghq.com/account_management/multi-factor_authentication/",
    "recovery": "https://docs.datadoghq.com/account_management/multi-factor_authentication/#mfa-recovery",
```

hkamran80 commented on img/d/datadoghq.com.svg:
```diff
@@ -0,0 +1,41 @@
+	c5.77-5.48-28.74-12.68-55.52,5.58c-19.75,13.47-20.38,42.35-1.47,58.72c1.89,1.62,3.45,2.77,4.91,3.71
+	c5.52-2.6,11.81-5.23,19.05-7.58c12.23-3.97,22.4-6.02,30.76-7.11c4-4.47,8.65-12.34,7.49-26.59
+	C401.49,268.05,386.84,271.12,397.85,260.65"/>
+</svg>
```
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 801 857"><path fill="#632ca6" d="m670 608-71-47-59 100-69-21-61 93 3 29 331-61-19-206-55 113zm-308-89 53-7c8 4 14 5 25 8 16 4 34 8 62-6 6-3 20-15 25-22l218-40 22 269-373 67-32-269zm404-97-22 4L703 0 0 82l87 702 82-11c-6-10-17-21-34-36-24-20-16-54-2-76 19-36 117-83 111-141-2-21-5-49-25-68l1 16s-8-10-12-24l-12-15c-3 9-2 18-2 18s-7-15-8-29c-4 6-5 18-5 18s-8-25-6-38c-4-12-16-35-13-87 22 15 69 12 87-16 6-9 11-33-3-81-8-31-30-77-38-95l-1 1c5 14 14 43 17 58 10 43 13 58 8 78-4 18-14 29-39 42s-59-18-61-20c-25-20-44-52-46-67-2-17 10-28 16-41l-19 6 26-22 16-11h-17s15-8 31-14h-22l60-26c18-8 36-6 46 9 13 19 26 29 56 35 17-8 23-12 45-18 20-21 36-24 36-24s-8 7-10 18c11-9 23-16 23-16l-9 15 1 1 29-14-10 13 38 1c46 1 55-49 73-55 22-8 32-12 69 24 32 32 58 89 45 101-10 11-31-4-54-32-12-15-21-33-25-56-4-19-18-30-18-30s8 18 8 34c0 9 1 42 15 60l-3 15c-17-19-52-34-58-38 20 16 64 53 81 88 17 33 7 63 15 71 3 2 35 43 41 63 11 35 1 73-13 96l-40 6c-6-2-10-2-15-6l9-20-3-4c-12 18-33 35-50 45a85 85 0 0 1-66 5c-48-15-93-47-104-56 0 0-1 7 1 9 13 13 40 38 67 55l-57 7 27 210-27 5c-12-41-34-68-58-83-21-14-51-17-79-12l-2 2c20-2 43 1 67 16 23 15 42 53 49 76 9 29 15 61-9 94-17 24-67 37-107 8 11 18 26 32 45 34 29 4 56-1 75-20 16-17 25-51 23-88l25-3 9 65 422-51-34-336zM509 245c-1 2-3 4 0 13v1l2 3c5 10 10 20 19 25l8-1c8 0 14 1 17 3v-8c0-13 3-35-22-47-9-4-23-3-27 3h2c7 3 2 5 1 8m70 121c-3-2-19-1-29 0-21 2-43 9-48 13-8 7-4 19 2 24 18 13 34 22 51 20 10-1 20-18 26-32 4-11 4-22-2-25M398 261c6-6-29-13-56 5a38 38 0 0 0-1 59l5 4a161 161 0 0 1 50-15c4-5 8-12 7-27-2-19-16-16-5-26"/></svg>
```

euan-reid committed:
Apply suggestions from code review

> If you go to the files tab and switch to the code view for SVG, it will show you the errors. What options did you run SVGOMG with?

The defaults plus multipass, as I recall, but looking at the output maybe I goofed and had prettify checked.

Co-authored-by: H. Kamran <hkamran@hkamran.com>
Signed-off-by: Euan Reid <github@euanreid.com>

euan-reid committed:
Update img/d/datadoghq.com.svg

Co-authored-by: H. Kamran <hkamran@hkamran.com>
Signed-off-by: Euan Reid <github@euanreid.com>
